keyboards/cannonkeys/rekt1800/bootloader_defs.h

```diff
@@ -0,0 +1,7 @@
+/* Address for jumping to bootloader on STM32 chips. */
```

Comment:
This file can be deleted, so as to use `drivers/boards/GENERIC_STM32_F072XB/bootloader_defs.h`.